packages/apputils-extension/src/statusbarplugin.ts

```diff
@@ -49,8 +50,25 @@ export const kernelStatus: JupyterFrontEndPlugin<IKernelStatusModel> = {
       await sessionDialogs.selectKernel(item.model.sessionContext);
     };
 
+    const changeKernelOnKeyDown = async (
+      event: KeyboardEvent<HTMLImageElement>
+    ) => {
+      if (
+        event.key === 'Enter' ||
+        event.key === 'Spacebar' ||
+        event.key === ' '
+      ) {
+        event.preventDefault();
+        event.stopPropagation();
+        changeKernel();
```

Comment:
This should either `return` or `await` the promise - otherwise lint will rightly fail.

```suggestion
        return changeKernel();
```